website_sale_product_attribute_value_filter_existing/tests/test_website_sale_product_attribute_value_filter_existing.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo.tests.common import HttpCase, tagged
+
+from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+
+
+@tagged("post_install", "-at_install")
+class WebsiteSaleHttpCase(HttpCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
+        # Models
+        ProductAttribute = cls.env["product.attribute"]
+        ProductAttributeValue = cls.env["product.attribute.value"]
+        ProductAttributeLine = cls.env["product.template.attribute.line"]
+        cls.product_attribute = ProductAttribute.create(
+            {"name": "Test Special Color", "create_variant": "no_variant"}
+        )
+        cls.product_attribute_value_red = ProductAttributeValue.create(
+            {"name": "Test red", "attribute_id": cls.product_attribute.id}
+        )
+        cls.product_attribute_value_green = ProductAttributeValue.create(
+            {"name": "Test green", "attribute_id": cls.product_attribute.id}
+        )
+        cls.product_attribute_value_blue = ProductAttributeValue.create(
+            {"name": "Test blue", "attribute_id": cls.product_attribute.id}
+        )
```

Comment:
The yellow attribute that is tested in the tour that is not present is not really present because it is not defined. That part needs to be corrected and the unused attribute needs to be added.

Reply:
https://github.com/OCA/e-commerce/pull/979